trxdb-loader/loader.go

```diff
@@ -374,10 +379,30 @@ func (l *TrxDBLoader) ShowProgress(blockNum uint64) {
 	now := time.Now()
 	if l.lastTickTime.Before(now.Add(-5 * time.Second)) {
 		if !l.lastTickTime.IsZero() {
+
+			blockSec := float64(blockNum-l.lastTickBlock) / float64(now.Sub(l.lastTickTime)/time.Second)
+
+			var totalBlockSec float64 = 0
```

Comment:
Same thing as before, we never set the value on variable declaration if we want the init(nil) value. The rest of codebase uses one of:
* `x := float(45)`
or
* `var x float` (since 0 is the init value) <-- use this one here and below